dll/win32/shell32/wine/shellpath.c

```diff
@@ -67,11 +67,11 @@ typedef enum _NT_PRODUCT_TYPE
 static BOOL
 DoGetProductType(PNT_PRODUCT_TYPE ProductType)
 {
-    static const WCHAR ProductOptions[] = L"System\\CurrentControlSet\\Control\\ProductOptions";
+    static const WCHAR ProductOptions[] = L"SYSTEM\\CurrentControlSet\\Control\\ProductOptions";
     HKEY hKey;
     LONG error;
     WCHAR szValue[32];
-    DWORD cbValue;
+    DWORD cbValue, dwType;
     static DWORD s_dwProductType = 0;
 
     if (s_dwProductType != 0)
```

Comment:
So this means that if someone during runtime (but not by killing the shell) changes the product type to something else, the checked type will remain the old one?